modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/JavaJAXRSSpecServerCodegen.java

```diff
@@ -134,18 +141,38 @@ public void processOpts() {
                     (sourceFolder + '/' + invokerPackage).replace(".", "/"), "RestApplication.java"));
         }
 
-        supportingFiles.add(new SupportingFile("openapi.mustache",
-                "src/main/openapi",
-                "openapi.yaml")
-        );
+        if(StringUtils.isNotEmpty(openApiSpecFileLocation)) {
+            int index = openApiSpecFileLocation.lastIndexOf('/');
```

Comment:
@jmini shall we use `File.separator` instead of `/` so that the option also works in Windows?

Reply:
I think even on windows the users will use `openApiSpecFileLocation = some/location/openapi.yaml` as input.
(think of a maven/gradle build that works on both windows and linux/macos)

That will result of creating a file at `output-folder\some\location\openapi.yaml`.

As you can see I have a test:
https://github.com/OpenAPITools/openapi-generator/blob/516163b509e5c15f18e6cdc3a9885291a7b30417/modules/openapi-generator/src/test/java/org/openapitools/codegen/java/jaxrs/JavaJAXRSSpecServerCodegenTest.java#L236-L255
Line 240 the variable is set to `"src/main/resources/META-INF/openapi.yaml"`
Line 255 there is a check that the file is created.

This is already working on windows (appveyor CI is green)

Reply:
> I think even on windows the users will use openApiSpecFileLocation = some/location/openapi.yaml as input.

👌 Let's see.